src/collection-facets.ts

```diff
@@ -489,6 +549,22 @@ export class CollectionFacets extends LitElement {
         opacity: 0.5;
       }
 
+      /* add the following styles to ensure proper modal visibility */
+      body.modal-manager-open {
+        overflow: hidden;
+      }
+      modal-manager {
```

Comment:
when moving `<modal-manager>` out of this component, the styles should probably live in demo's `<app-root>` as well as offshoot's `<app-root>`

Reply:
if you add a class to <modal-manager> after loading config, like:
`modalManger.setAttribute('class', 'more-search-facets');`

so you can write the css scoped like:
`modal-manager.more-search-facets { ... }`

but remember to remove the class attribute when finished w modal (close)